lib/plugins/authplain/lang/nl/lang.php

```diff
@@ -2,9 +2,11 @@
 
 /**
  * @license    GPL 2 (http://www.gnu.org/licenses/gpl.html)
- * 
+ *
  * @author Hugo Smet <hugo.smet@scarlet.be>
+ * @author Sjoerd <sjoerd@sjomar.eu>
  */
 $lang['userexists']            = 'Er bestaat al een gebruiker met deze loginnaam.';
 $lang['usernotexists']         = 'Sorry, deze gebruiker bestaat niet.';
 $lang['writefail']             = 'Onmogelijk om de gebruikers data te wijzigen. Gelieve de Wiki-Admin te informeren.';
+$lang['protected']             = 'Gegevens van gebruiker %s is beschermd en kan niet worden veranderd of verwijderd';
```

Comment:
"Gegevens" is plural so this should be: `Gegevens van gebruiker %s zijn beschermd en kunnen niet worden veranderd of verwijderd`